public/assets/scripts/app.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @preserve Copyright 2017. Sean Bailey, All Rights Reserved,
+ *
+ * @author Sean Bailey, sean@seanbailey.io
+ */
+
+
+
+/**
+ * A function which fires when DOM has loaded. Essentially a vanilla version of
+ * JQuery's $(document).ready().
+ *
+ * @param {!Object} func Function to fire when DOM is ready.
+ */
+function ready (func) {
+    if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading") {
+        func();
+    } else {
+        document.addEventListener("DOMContentLoaded", func);
+    }
+}
+
+
+
+/**
+ * Adjusts the text area's vertical size according to its content.
+ *
+ * @param {!Object} elem Text area element to resize.
+ */
+function resizeTextArea (elem) {
+    elem.style.height = (elem.scrollHeight) + "px";
```

Comment:
I think there should be a min height so in normal cases, textareas can be roundly equals between 3 parts.

Reply:
Each of the textarea's actually have `row="1"`, which causes them to default to the height of 1 line, including padding.